<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LTSM_HelperWindow</class>
 <widget class="QMainWindow" name="LTSM_HelperWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>233</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>LTSM_HelperWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(20, 20, 70);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>X11 Remote Desktop</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout5">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelLogo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ltsm_helper.qrc">:/ltsm/ltsm_logo.png</pixmap>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout2">
          <item>
           <widget class="QLabel" name="labelUsername">
            <property name="text">
             <string>username:</string>
            </property>
            <property name="buddy">
             <cstring>comboBoxUsername</cstring>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout3">
          <item>
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>password:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditPassword</cstring>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>22</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelEncryption">
            <property name="text">
             <string>encryption:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBoxUsername">
          <property name="inputMethodHints">
           <set>Qt::ImhLatinOnly|Qt::ImhLowercaseOnly</set>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditEncryption">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>none</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonLogin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>22</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelInfo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSessions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>default</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelXkb">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBoxUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pushButtonLogin</tabstop>
  <tabstop>comboBoxSessions</tabstop>
 </tabstops>
 <resources>
  <include location="ltsm_helper.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBoxUsername</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>LTSM_HelperWindow</receiver>
   <slot>usernameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLogin</sender>
   <signal>clicked()</signal>
   <receiver>LTSM_HelperWindow</receiver>
   <slot>loginClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LTSM_HelperWindow</receiver>
   <slot>passwordChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>usernameChanged(QString)</slot>
  <slot>loginClicked()</slot>
  <slot>passwordChanged(QString)</slot>
  <slot>titlePressed()</slot>
  <slot>titleReleased()</slot>
 </slots>
</ui>
